driver-mapping/src/main/java/com/datastax/driver/mapping/Mapper.java

```diff
@@ -1028,29 +1090,37 @@ boolean isIncludedInQuery() {
             }
 
             @Override
-            void appendTo(Insert.Options usings) {
-                throw new UnsupportedOperationException("shouldn't be called");
+            void validate(QueryType qt, MappingManager manager) {
+                checkArgument(qt == QueryType.SAVE || qt == QueryType.DEL || qt == QueryType.GET,
```

Comment:
Just noticed: this check is useless, these is not other `QueryType` at the moment.